libs/utils/src/lib/spreadsheet/import.ts

```diff
@@ -241,7 +242,28 @@ export function getStaticList(scope: Scope, value: string, separator:string, nam
   return values;
 }
 
-export function split(cell: string, separator:string) {
+export function getGroupedList<GS extends GroupScope>(value: string, groupScope: GS, separator: string) {
+  const elements = split(value, separator);
+  const groupLabels = staticGroups[groupScope].map(group => group.label);
+  let allElements = elements.map(element => {
+    if (groupLabels.includes(element)) {
+      const groups = staticGroups[groupScope] as StaticGroup<GS>[]
+      return groups.find(group => group.label === element).items;
+    }
+    return [element];
+  }).flat()
+
+  allElements = Array.from(new Set(allElements));
+  const elementList = allElements.join(separator);
```

Comment:
Avoid using `let` when possible. For example doing that would avoid the need of `let` above.
```suggestion
  const elementList = Array.from(new Set(allElements)).join(separator);
```